tests/unit/setup-globals.js

```diff
@@ -24,3 +24,26 @@ global.wp = {
 		},
 	},
 };
+
+global.ampStoriesFonts = [
+	{
+		name: 'Arial',
+		slug: 'arial',
+	},
+	{
+		name: 'Roboto',
+		slug: 'roboto',
+		handle: 'roboto-font',
+		src: 'https://fonts.googleapis.com/css?family=Roboto%3A400%2C700&subset=latin%2Clatin-ext',
```

Comment:
Aside: we should consider using `display` at some point for fonts used in stories.

Reply:
Created #2367 to track this.